ext/libgecode3/extconf.rb

```diff
@@ -81,6 +98,14 @@ def self.patch_configure
     end
   end
 
+  def self.make
+    if openbsd?
+      "gmake"
+    else
+      "make"
+    end
+  end
```

Comment:
this should probably be changed to prefer gmake over make if its found in the path.